pdal/util/FileUtils.hpp

```diff
@@ -302,8 +313,8 @@ namespace FileUtils
       \return  MapContext.  addr() gets the mapped address.  what() gets
          any error message.  addr() returns nullptr on error.
     */
-    PDAL_DLL MapContext mapFile(const std::string& filename, bool readOnly,
-        size_t pos, size_t size);
+    PDAL_DLL MapContext mapFile(const std::string& filename, bool readOnly = true,
```

Comment:
Is this API change going to bother anyone? Probably not, but we should include a note in the release notes.

Reply:
The API is compatible. uintmax_t is at least as bit as size_t.